tenseal/cpp/context/tensealcontext.cpp

```diff
@@ -118,9 +118,16 @@ shared_ptr<TenSEALContext> TenSEALContext::Create(
     return shared_ptr<TenSEALContext>(new TenSEALContext(input, n_threads));
 }
 
+bool TenSEALContext::has_public_key() const {
+    return !(this->_public_key == nullptr);
```

Comment:
I think `return this->_public_key != nullptr;` is more readable.
same for others.